controllers/collect_vision_data/collect_vision_data.cpp

```diff
@@ -91,6 +91,15 @@ int main(int argc, char** argv) {
     right_camera->enableRecognitionSegmentation();
     left_camera->enableRecognitionSegmentation();
 
+    int camera_width = left_camera->getWidth();
+    int camera_height = left_camera->getHeight();
+    double horizontal_fov = left_camera->getFov();
+    double vertical_fov = 2 * std::atan(std::tan(horizontal_fov * 0.5) * ((double) camera_height / camera_width));
```

Comment:
Need some clang tidy in here to protect you from these C style casts!
```suggestion
    double vertical_fov = 2 * std::atan(std::tan(horizontal_fov * 0.5) * (double(camera_height) / camera_width));
```